spec/helpers/application_helper_spec.rb

```diff
@@ -311,22 +317,58 @@
 
   describe 'expire_date_label_and_value' do
 
-    it 'returns date with style "yes" if expire_date more than a month away' do
-      user_payment.update(expire_date: Time.zone.today + 2.months)
-      response = /class="Yes".*#{user_payment.expire_date}/
-      expect(expire_date_label_and_value(user)).to match response
-    end
+    context 'user' do
 
-    it 'returns date with style "maybe" if expire_date less than a month away' do
-      user_payment.update(expire_date: Time.zone.today + 2.days)
-      response = /class="Maybe".*#{user_payment.expire_date}/
-      expect(expire_date_label_and_value(user)).to match response
+      it 'returns date with style "yes" if expire_date more than a month away' do
+        user_payment.update(expire_date: Time.zone.today + 2.months)
+        response = /class="Yes".*#{user_payment.expire_date}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
+
+      it 'returns date with style "maybe" if expire_date less than a month away' do
+        user_payment.update(expire_date: Time.zone.today + 2.days)
+        response = /class="Maybe".*#{user_payment.expire_date}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
+
+      it 'returns date with style "no" if expired' do
+        user_payment.update(expire_date: Time.zone.today - 1.day)
+        response = /class="No".*#{user_payment.expire_date}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
+
+      it 'returns tooltip explaining expiration date' do
+        user_payment.update(expire_date: Time.zone.today)
+        response = /#{t('users.show.membership_expire_date_tooltip')}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
     end
 
-    it 'returns date with style "no" if expired' do
-      user_payment.update(expire_date: Time.zone.today - 1.day)
-      response = /class="No".*#{user_payment.expire_date}/
-      expect(expire_date_label_and_value(user)).to match response
+    context 'company' do
+
+      it 'returns date with style "yes" if expire_date more than a month away' do
+        brand_payment.update(expire_date: Time.zone.today + 2.months)
+        response = /class="Yes".*#{brand_payment.expire_date}/
+        expect(expire_date_label_and_value(company)).to match response
+      end
+
+      it 'returns date with style "maybe" if expire_date less than a month away' do
+        brand_payment.update(expire_date: Time.zone.today + 2.days)
```

Comment:
your `it...` statement specifies "less than a month" but the time that you're actually setting is just +2.days
Should have the date that you're using as close as possible to what you've described in the "it..." statement.